modules/albo-installation.adoc

```diff
@@ -53,236 +52,173 @@ $ cat <<EOF > "${SCRATCH}/trust-policy.json"
    }
  },
  "Principal": {
-   "Federated": "arn:aws:iam::$AWS_ACCOUNT_ID:oidc-provider/${OIDC_ENDPOINT}"
+   "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_ENDPOINT}"
  },
  "Action": "sts:AssumeRoleWithWebIdentity"
  }
  ]
 }
 EOF
 ----
-+
-. Create an AWS IAM role for the AWS Load Balancer Operator:
+.. Create the Operator role using the trust policy:
 +
 [source,terminal]
 ----
-$ ROLE_ARN=$(aws iam create-role --role-name "${ROSA_CLUSTER_NAME}-alb-operator" \
-   --assume-role-policy-document "file://${SCRATCH}/trust-policy.json" \
-   --query Role.Arn --output text)
-$ echo $ROLE_ARN
-
-$ aws iam attach-role-policy --role-name "${ROSA_CLUSTER_NAME}-alb-operator" \
-     --policy-arn $POLICY_ARN
+$ aws iam create-role --role-name "${CLUSTER_NAME}-alb-operator" \
+    --assume-role-policy-document "file://${SCRATCH}/operator-trust-policy.json"
 ----
 +
-. Create a secret for the AWS Load Balancer Operator to assume our newly created AWS IAM role:
+Take note of the Operator role ARN in the output. This is referred to as the `$OPERATOR_ROLE_ARN` for the remainder of this process.
```

Comment:
I recommend explicitly setting the OPERATOR_ROLE_ARN here, either via something like this:

Export the OPERATOR_ROLE_ARN variable:

```
export OPERATOR_ROLE_ARN=$your_role_arn
```

Or in the create role command like was originally there:

```
OPERATOR_ROLE_ARN=$(aws iam create-role --role-name "${CLUSTER_NAME}-alb-operator" \
  --assume-role-policy-document "file://${SCRATCH}/operator-trust-policy.json" \
  --query Role.Arn --output text)

echo ${OPERATOR_ROLE_ARN}
```

Otherwise the user may proceed to installing the operator later without ever checking that this variable was set and being confused as to why it isn't working